eng/Versions.props

```diff
@@ -7,93 +7,6 @@
     <!-- Opt-out repo features -->
     <UsingToolXliff>false</UsingToolXliff>
     <!-- Libs -->
-    <CommandLineParserVersion>2.2.1</CommandLineParserVersion>
-    <CredentialManagementVersion>1.0.2</CredentialManagementVersion>
-    <HandlebarsNetVersion>1.10.1</HandlebarsNetVersion>
-    <LibGit2SharpVersion>0.25.2</LibGit2SharpVersion>
-    <log4netVersion>2.0.10</log4netVersion>
-    <SystemNetHttpVersion>4.3.4</SystemNetHttpVersion>
-    <AzureCoreVersion>1.25.0</AzureCoreVersion>
-    <AzureIdentityVersion>1.8.0</AzureIdentityVersion>
-    <AzureSecurityKeyVaultSecretsVersion>4.4.0</AzureSecurityKeyVaultSecretsVersion>
-    <AzureStorageBlobsVersion>12.13.0</AzureStorageBlobsVersion>
-    <AzureDataTablesVersion>12.5.0</AzureDataTablesVersion>
-    <FluentAssertionsVersion>5.10.3</FluentAssertionsVersion>
-    <NETStandardLibraryVersion>1.6.1</NETStandardLibraryVersion>
-    <MicrosoftApplicationInsightsVersion>2.16.0</MicrosoftApplicationInsightsVersion>
-    <MicrosoftAzureServicesAppAuthenticationVersion>1.3.1</MicrosoftAzureServicesAppAuthenticationVersion>
-    <MicrosoftDataAnalysisVersion>0.1.0</MicrosoftDataAnalysisVersion>
-    <MicrosoftDataODataVersion>5.8.4</MicrosoftDataODataVersion>
-    <MicrosoftDataServicesClientVersion>5.8.4</MicrosoftDataServicesClientVersion>
-    <MicrosoftBuildVersion>15.7.179</MicrosoftBuildVersion>
-    <MicrosoftBuildFrameworkVersion>15.7.179</MicrosoftBuildFrameworkVersion>
-    <MicrosoftBuildTasksCoreVersion>15.7.179</MicrosoftBuildTasksCoreVersion>
-    <MicrosoftBuildUtilitiesCoreVersion>15.7.179</MicrosoftBuildUtilitiesCoreVersion>
-    <MicrosoftCodeAnalysisAnalyzersVersion>2.6.3</MicrosoftCodeAnalysisAnalyzersVersion>
-    <MicrosoftCodeAnalysisCSharpVersion>2.9.0</MicrosoftCodeAnalysisCSharpVersion>
-    <MsbuildTaskMicrosoftCodeAnalysisCSharpVersion>3.8.0</MsbuildTaskMicrosoftCodeAnalysisCSharpVersion>
-    <GenAPIMicrosoftCodeAnalysisVersion>4.3.0</GenAPIMicrosoftCodeAnalysisVersion>
     <GenAPISystemCommandLineVersion>2.0.0-beta4.22272.1</GenAPISystemCommandLineVersion>
```

Comment:
@dtivel This version is referenced in Directory.Packages.props. Any reason why this is used instead of a dependency on System.CommandLine?

Reply:
@mmitche, I figured the version would get updated for free.  I take it https://github.com/dotnet/sign/blob/a68e3d6b26615ce9db9b6087fa2a3e4c7a7a495f/Directory.Packages.props#L26 should reference the version explicitly and independently of eng/Versions.props?

Reply:
You have this correct in eng/Versions.props and Directory.Packages.props, except that `GenAPISystemCommandLineVersion` doesn't match any dependency in eng/Version.Details.xml. It should be `SystemCommandLineVersion`, and the eng/Version.Details.xml would have a corresponding entry for that dependency.

Reply:
BTW, Sign.Cli _does_ have a [package reference for System.CommandLine](https://github.com/dotnet/sign/blob/main/src/Sign.Cli/Sign.Cli.csproj#L18).  It's just that the package version is controlled through that property.  Is the property not intended for this purpose?

Reply:
The property isn't correctly named for that, if you also want automated updates via Maestro.

Reply:
I see.  I'll just use an explicit version.  I'll create a separate PR for that.  I don't think it's blocking this PR.